### DISCUSSION TITLE
How to access MetroLog.Maui logger from within static classes

### DISCUSSION THREAD

**Discussion by Orgbrat** [roubachof/MetroLog#27] [Q&A]
I have a Maui application that I setup The MetroLog.Maui logging as was shown in this repo Sample application. But I have a very few static classes that I also need to use the logging from. Of course I have used DI every where else in all other classes but this does not work in static classes. What is the best way use the MetroLog logger in this kind of instance?

**Comment by roubachof** [comment]
Use the LoggerFactory with standard configuration (it's at the beginning of the readme file)

**Reply by Orgbrat** [reply]
@roubachof  Does that break all the DI code already added thru constructor DI or will it continue working as expected?

**Reply by roubachof** [reply]
you *could* create both configuration. It's a bit weird but it should work.
